Tests/TuistDependenciesAcceptanceTests/DependenciesAcceptanceTests.swift

```diff
@@ -5,29 +5,40 @@ import TuistSupportTesting
 import XcodeProj
 import XCTest
 
-final class DependenciesAcceptanceTestAppWithSPMDependencies: TuistAcceptanceTestCase {
-    func test_app_spm_dependencies() async throws {
-        try setUpFixture(.appWithSpmDependencies)
+/* final class DependenciesAcceptanceTestAppWithSPMDependencies: TuistAcceptanceTestCase {
+     func test_app_spm_dependencies() async throws {
+         try setUpFixture(.appWithSpmDependencies)
+         try await run(InstallCommand.self)
+         try await run(GenerateCommand.self)
+         try await run(BuildCommand.self, "App")
+         try await run(BuildCommand.self, "VisionOSApp")
+         try await run(TestCommand.self, "AppKit")
+     }
+ }
+ */
+final class DependenciesAcceptanceTestIosAppWithSPMDependencies: TuistAcceptanceTestCase {
+    func test_ios_app_spm_dependencies() async throws {
+        try setUpFixture(.iosAppWithSpmDependencies)
         try await run(InstallCommand.self)
         try await run(GenerateCommand.self)
         try await run(BuildCommand.self, "App")
-        try await run(BuildCommand.self, "VisionOSApp")
-        try await run(TestCommand.self, "AppKit")
+        try await run(TestCommand.self, "App")
     }
 }
 
-final class DependenciesAcceptanceTestAppWithSPMDependenciesWithoutInstall: TuistAcceptanceTestCase {
-    func test() async throws {
-        try setUpFixture(.appWithSpmDependencies)
-        do {
-            try await run(GenerateCommand.self)
-        } catch {
-            XCTAssertEqual(
-                (error as? FatalError)?.description,
-                "We could not find external dependencies. Run `tuist install` before you continue."
-            )
-            return
-        }
-        XCTFail("Generate should have failed.")
-    }
-}
+/* final class DependenciesAcceptanceTestAppWithSPMDependenciesWithoutInstall: TuistAcceptanceTestCase {
+     func test() async throws {
+         try setUpFixture(.appWithSpmDependencies)
+         do {
+             try await run(GenerateCommand.self)
+         } catch {
+             XCTAssertEqual(
+                 (error as? FatalError)?.description,
+                 "We could not find external dependencies. Run `tuist install` before you continue."
+             )
+             return
+         }
+         XCTFail("Generate should have failed.")
+     }
+ }
+ */
```

Comment:
Why is this commented out?

Reply:
I was just trying to figure out why are tests failing for KSCrash on CI. I'll make this PR draft for now and come back to it once [this](https://github.com/tuist/tuist/pull/6264) is merged.